e2e/version/version.go

```diff
@@ -96,6 +96,19 @@ func (c *ctx) testEqualVersion(t *testing.T) {
 	}
 }
 
+// Test the help option
+func (c *ctx) testHelpOption(t *testing.T) {
+	c.env.RunSingularity(
+		t,
+		e2e.WithCommand("version"),
+		e2e.WithArgs("--help"),
+		e2e.ExpectExit(
+			0,
+			e2e.ExpectOutput(e2e.ContainMatch, "Show the version for Singularity"),
```

Comment:
Does this only check the first line of the output?

Reply:
Good question... I will investigate.

Reply:
No, this will check the entire output for `"Show the version for Singularity"`

Reply:
I will switch to `RegexMatch` after a suggestion from @ikaneshiro